iOS/MAX Unreal Plugin.xcodeproj/xcuserdata/vedant.mehta.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
No need to check this file in since it's local to your workspace - feel free to delete
<img width="245" alt="image" src="https://user-images.githubusercontent.com/5104410/165595504-791bdda8-024d-48c0-a277-c9bfd2f6f0d4.png">